src/js/components/TextInput/doc.js

```diff
@@ -101,7 +101,8 @@ Only use this when the containing context provides sufficient affordance`,
       PropTypes.oneOfType([
         PropTypes.shape({
           label: PropTypes.node,
-          value: PropTypes.any,
+          // eslint-disable-next-line
+          value: PropTypes.any, // this is actually any
```

Comment:
this isn't immediately clear to me... if this is saying that we have to disable eslint because the violation is that we can't have an `any` type, because this is intentionally type of `any`, then could we add that rule to the the disable comment? At first i was thinking that the violation was having inline comments lol Maybe could use "intentional" vs "actually"?